Sources/AMQPClient/AMQPConnection.swift

```diff
@@ -47,46 +50,124 @@ internal final class AMQPConnection {
         return self.channel.isActive && self.state.isConnected
     }
 
-    var closeFuture: NIOCore.EventLoopFuture<Void> {
+    public var closeFuture: NIOCore.EventLoopFuture<Void> {
         get { return  self.channel.closeFuture }
     }
 
-    init(channel: NIOCore.Channel) {
+    private var channels = AMQPChannels()
+
+    init(channel: NIOCore.Channel, multiplexer: AMQPConnectionMultiplexHandler, channelMax: UInt16) {
         self.channel = channel
+        self.multiplexer = multiplexer
+        self.channelMax = channelMax
     }
 
-    static func create(use eventLoopGroup: EventLoopGroup, from config: AMQPClientConfiguration) -> EventLoopFuture<AMQPConnection> {
-        return self.boostrapChannel(use: eventLoopGroup, from: config)
-            .map { AMQPConnection(channel: $0) }
+    /// Connect to broker.
+    /// - Parameters:
+    ///     - eventLoop: EventLoop on which to conntec.
+    ///     - config: Configuration
+    /// - Returns:  EventLoopFuture with Connection object.
+    public static func connect(use eventLoop: EventLoop, from config: AMQPConnectionConfiguration) -> EventLoopFuture<AMQPConnection> {
+        let multiplexer = AMQPConnectionMultiplexHandler(config: config.server)
+
+        return self.boostrapChannel(use: eventLoop, from: config, with: multiplexer)
+            .flatMap { channel in
+                multiplexer.start(initialSequence: PROTOCOL_START_0_9_1)
```

Comment:
the start should be triggered inside the handler. Observe `channelActive` inside the ChannelHandler to trigger the start handshake.

Reply:
moved to channelActive